ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/DataStructureUtil.java

```diff
@@ -217,7 +208,7 @@ public static AttestationData randomAttestationData(int seed) {
   }
 
   public static AttestationData randomAttestationData() {
-    return randomAttestationData(randomInt());
+    return randomAttestationData(randomInt(12344));
```

Comment:
This function is supposed to be non-deterministic. You sure you want to make it deterministic?

Reply:
I really do want to get rid of all the randomness in tests but probably shouldn't do it in this PR so have backed this bit out again.

Reply:
@ajsutton we did remove the non-seeded randomness.  There is a specific randomAttestationData(int seed) method. 

```
  public static AttestationData randomAttestationData(int seed) {
    return new AttestationData(
        randomBytes32(seed),
        randomCheckpoint(seed++),
        randomCheckpoint(seed++),
        randomCrosslink(seed++));
  }

  public static AttestationData randomAttestationData() {
    return randomAttestationData(randomInt(12344));
  }
```